design-at-cornell/src/events/dashboard/EventDisplay.tsx

```diff
@@ -1,57 +1,32 @@
 import React from 'react';
-import { EventContainer, Settings, TagsContainer, Tag, Link } from '../EventsStyles';
-import { Event } from '../Events';
+import { EventContainer, Settings, EventLink } from '../EventsStyles';
+import { Event } from '../../../../server/types';
 import time from '../../static/images/time.svg';
 import loc from '../../static/images/location.svg';
-import { buttons } from '../../constants/eventButtons';
+import linkIcon from '../../static/images/link-icon.svg';
 
 const EventDisplay = (event: Event) => (
   <EventContainer>
-    <h1>{event.name}</h1>
+    <h1>{event.title}</h1>
     <Settings>
       <img src={time} alt="time" />
-      <p>{formatRange(event.startTime, event.endTime)}</p>
+      <p>
+        {event.date.substring(0, 10) +
+          ' ' +
+          event.date.substring(11, 16) +
+          event.date.substring(19) +
+          ' EST'}
+      </p>
       <img src={loc} alt="location" />
       <p>{event.location}</p>
```

Comment:
There are some events that do not have any location information. Can you modify the code so that the location icon only shows when there is actually a location associated with the event?